k8s/overlays/nerc-ocp-infra/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+bases:
```

Comment:
`bases` has been deprecated for a while: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/bases/

it should be changed to `resources`.

Reply:
ok changed bases to resources.  Had to update my openshift client (oc) in order to support this change.